enhancements/kueue/openshift-kueue.md

```diff
@@ -0,0 +1,758 @@
+
+Yes. Standalone clusters will be able to install Kueue from OperatorHub.
+
+#### Single-node Deployments or MicroShift
+
+##### Microshift
+
+To support microshift, one would need to install OLM.
+Once OLM is enabled, one can install Cert Manager and the operator.
+After that, the operator will also require that metrics be disabled. We will include a special flag for rare cases 
+where customers would disable metrics.
+
+Microshift does not enable openshift-monitoring so metrics would be disabled in this case.
+
+##### Single-node Deployments
+
+I don't forsee any issue with SNO.
+
+### Implementation Details/Notes/Constraints
+
+Cert Manager will be used to manage certificates so our operator will have a hard dependency on Cert Manager.
+
+#### GA Feature Statement
+
+##### Supported Features
+
+In Tech preview, we will provide the following features:
+
+- [Preemption](https://kueue.sigs.k8s.io/docs/concepts/preemption/)
+- Advanced Resource management: Comprising: resource flavor fungibility, fair sharing, cohorts and preemption with a variety of policies between different tenants.
+- Support for GPU model servers via pod-integration
+- Namespace opt in for frameworks.
+- Metrics for Kueue in OCP monitoring.
+- [Workload Priority Classes](https://kueue.sigs.k8s.io/docs/concepts/workload_priority_class/)
+- LocalQueue, ClusterQueues, Workloads, Worklow Priorites, Admission Checks are all supported.
+- Support for deployments, pods, statefulsets.
+- [Use of resource flavors to describe heterogeous clusters](https://kueue.sigs.k8s.io/docs/concepts/resource_flavor/)
+- Use of ManagedJobsWithoutQueueName
+- Gang admission via `WaitForPodsReady` Kueue configurations.
+- Fairsharing
+
+##### Not Supported for initial GA
+
+We will not provide the following features for our 1.0 release.
+
+- MultiKueue
+- Autoscaling
+- TopologyAwareScheduling
+- Resource Transformations
+- KueueViz
+- [VisibilityOnDemand](https://kueue.sigs.k8s.io/docs/tasks/manage/monitor_pending_workloads/pending_workloads_on_demand/)
+- Topology CRD (Topology Aware Scheduling)
+- Cohort CRD (hierachial queueing)
+
+##### Feature support post GA
+
+Each of these features are valid. We will add support for them as dedicated RFE.
+Our focus for initial phase will be the supported features above.
+
+#### Release Schedule
+
+
+| Kueue Operator     |  Stage       |  OCP Version   |  Kueue   |
+| ------------------ | -------      | ---------------| -------- |
+| 0.1                | GA Candidate | 4-17-4.19      | 0.11.z   |
+| 1.0                | GA           | 4-17-4.19      | 0.11.z   |
+
+Kueue releases 6 times a year, roughly. 
+They will have 2 releases per k8s version so we can take the latest version that is built with the kubernetes version that OCP comes with.
+
+There will be no tech preview for this operator and we will go to a GA release.
+
+#### Stability of APIs
+
+A goal will be to engage with upstream to promote APIs to v1 and aim to graduate these APIs to stable.
```

Comment:
Would be good to explain the maintenance that will be needed as a result of shipping something that isn't v1 (future upgrades of the API, conversion webhooks, eventual removal of the old API versions)

Reply:
>Given that we are releasing this on July 10th, is this necessary to call out?
> Sounds like we accept this risk.

I'm not sure I would accept a risk without first understanding what that risk is 😉